content/docs/central-dash/_index.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
+weight = 35
++++
+
+Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
```

Comment:
Remove the content from this page. The docs follow the pattern of having an empty `_index.md` file for each section, apart from the front matter at the top of the page, apart from the front matter at the top surrounded by the plus signs. The empty index page causes the doc site to generate a list of pages in the section. See the pattern here: https://raw.githubusercontent.com/kubeflow/website/master/content/docs/fairing/_index.md
which corresponds to this page:
https://www.kubeflow.org/docs/fairing/

(Also, the content here is a duplicate of your page `central-dashboard.md` in this PR.)